core/src/main/java/de/danoeh/antennapod/core/ClientConfig.java

```diff
@@ -41,6 +42,7 @@ public static synchronized void initialize(Context context) {
         UpdateManager.init(context);
         PlaybackPreferences.init(context);
         NetworkUtils.init(context);
+        CastManager.init(context);
```

Comment:
That's a good sign. Yes, I'm checking for play services when the user tries to change the settings, but I should review the code/documentation on the API and make sure that the 'dormant' components won't give any problem.